### PR TITLE
fix for geogebra.org

### PR DESCRIPTION
**Pull request by xxyzz**
https://www.geogebra.org/graphing
Before:
Screen Shot 2020-05-16 at 07 42 49
Screen Shot 2020-05-16 at 07 46 57

After:
Screen Shot 2020-05-16 at 07 39 05
Screen Shot 2020-05-16 at 07 47 40

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -1260,6 +1260,19 @@ img.listview__image, img.galleryview__image {
...
+
+INVERT
+.icon-m
+.elemText canvas
+.gwt-Image
+.gwt-StackPanelItem img
+.GeoGebraMenuImage.menuImg
+.menuImg
+.buttonContent.stylebarButton
+
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:1273]: Let's add `canvas[class^="cursor"]` So the canvas will also be targetted

Comment (xxyzz) [reply src/config/dynamic-theme-fixes.config:1273]: I tried to invert the main canvas but the lines were blinking when I moved around. The app supports change lines and background color in settings, that works well.

Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:1273]: I saw that when trying to just get the class `cursor_hit`
But when blinking it just changes the class after the _ so cursor_drag cursor_pen
So with CSS3 selectors we can target `startWith` with ^= so
canvas[class^="cursor"] means that it only target canvas with a class that start with cursor and then you don't have the blinking anymore :)

Comment (xxyzz) [reply src/config/dynamic-theme-fixes.config:1273]: I used the selector `.EuclidianPanel canvas` earlier, I think the results are the same. After adding a few functions, you will notice the blinking at some part of the lines you added when you move the mouse or drag the canvas.

Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:1273]: .EuclidianPanel > canvas
Add that and then it will be merged

**Pull request merged**